src/Containers/App/App.test.js

```diff
@@ -1,21 +1,50 @@
-import React, { Component } from 'react'
-import App from './App'
+import React from 'react'
+import { App, mapStateToProps, mapDispatchToProps } from './App'
 import { shallow } from 'enzyme'
 import { fetchNotes } from '../../thunks/fetchNotes'
-
+jest.mock('../../thunks/fetchNotes')
 
 describe('App', () => {
   let wrapper
-
+  let fetchNotesMock
   beforeEach(() => {
-    wrapper = shallow(<App />)
+    fetchNotesMock = jest.fn()
+    wrapper = shallow(<App fetchNotes={fetchNotesMock} />)
```

Comment:
I like the way you implemented testing for a thunk to be called here. This is a great example for me to use for the NoteForm testing.